### DISCUSSION TITLE
Add Fluent Forms support

### DISCUSSION THREAD

**Discussion by LeoOosterloo** [pronamic/wp-pronamic-pay#357] [Ideas]
Just requested via email:

> I am looking into your solution for supporting payments and in particular the solutions for connecting with Mollie (iDeal, Credit Card).
> 
> I see you are supporting various Forms solutions like Gravity Forms, Ninja Forms......but we are currently using WPManagerNinja = Fluent Forms https://fluentforms.com/
> 
> Just asking if you integrate with the Fluent Forms solution or if you have plans for that.

**Comment by remcotolsma** [comment]
The previous request was received at info@pronamic.nl, @LeoOosterloo responded as follows (in Dutch):

> We ondersteunen op dit moment geen Fluent Forms en hebben op dit moment ook geen plannen om deze wel te gaan ondersteunen.
> Ik heb hem wel op ons lijstje met 'feature requests' gezet, maar daar kan ik verder niets concreets over zeggen.

We just received a the following question:

> Ik maak nu gebruik van GF forms, is er ook een integratie met Fluentforms?

Internal HelpScout ticket: https://secure.helpscout.net/conversation/2299059311/25883/

I asked why they would want to switch from Gravity Forms to Fluent Forms.

**Comment by remcotolsma** [comment]
Response from customer:

> Ik gebruik al jaren gf forms, op zicht is het een hele goeie plugin, bijna nooit porblemen gehad. Ik heb fluent forms getest paar weken, het blaast gf forms en formidable weg. Kwa prijs verschilt ook niet veel want je hebt ook een gratis versie ervan. Webhook en airtable, crm, paypal, stripe, mollie etc etc zit allemaal inbegrepen bij pro addon. Kwa snelheid is fluent ook beter.
> 
> Bv voor gf forms moet ik extra addons kopen, bv voor webhook wat ik vaak gebruik en ook voor airtable betaal ik aan third party jaarlijks licentie. Bij fluent zit er allemaal inbegrepen. Support is ook beter bij hun.

Is it time to tackle the following issue/idea?

- https://github.com/pronamic/pronamic-pay/issues/48
